### PR TITLE
Update runner node dependencies

### PR DESCRIPTION
**Pull request by bameyrick**
### What is the context of this PR?
- Replace gulp eslint with eslint-cli, updated eslint configs
- Removed unused cypress config
- Removed gulp json formatter and replaced with prettier
- Removed browsersync and replaced with livereload
- Updated `make dev-compose-up` to pull latest version of launcher
- Updated `yarn start` to run `make dev-compose-up`, `make run`, and `livereload`
- Added .nvmrc
- Removed unused dependencies
- Removed unused variables and code that linter complained about

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Update runner node dependencies](https://trello.com/c/9iB25pE4/3060-update-runner-node-dependencies)

**Review by JamesGardiner — APPROVED** [review]

File: app/setup.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
         'https://fonts.googleapis.com',
         "'unsafe-inline'",
     ],
-    'connect-src': ["'self'", 'https://cdn.ons.gov.uk'],
+    'connect-src': ["'self'", 'ws://localhost:35729', 'https://cdn.ons.gov.uk'],
```
Comment (iwootten) [inline app/setup.py]: Is this for live reload? We probably should only be enabling it through a configuration as per setup_secure_headers does with the EQ_ENABLE_LIVE_RELOAD flag?

**Review by iwootten — DISMISSED** [review]

File: app/setup.py (hunk 1 above)
Comment (bameyrick) [reply app/setup.py]: Yeah it's for live reload... Could you give me a hand with that?

Comment (iwootten) [reply app/setup.py]: Think you should just be able to add it here, given the flag already exists: https://github.com/ONSdigital/eq-survey-runner/blob/0552d46048dcd77deff59b4f833a03d1cf39821f/app/setup.py#L236

Comment (bameyrick) [reply app/setup.py]: Cheers, updated this now

**Review by iwootten — APPROVED** [review]

**Comment by iwootten** [comment]
Just needs a squash after https://github.com/ONSdigital/eq-survey-runner/pull/2155 goes in.

**Pull request merged**